modules/core/src/passes/layers-pass.js

```diff
@@ -270,13 +278,20 @@ export function layerIndexResolver(startIndex = 0, layerIndices = {}) {
 }
 
 // Convert viewport top-left CSS coordinates to bottom up WebGL coordinates
-function getGLViewport(gl, {viewport}) {
-  // TODO - dummy default for node
+function getGLViewport(gl, {moduleParameters, target, viewport}) {
+  const useTarget = target && target.id !== 'default-framebuffer';
+  const pixelRatio =
+    (moduleParameters && moduleParameters.devicePixelRatio) || cssToDeviceRatio(gl);
+  const targetHeight = target ? target.height / pixelRatio : 100;
+
   // Fallback to width/height when clientWidth/clientHeight are 0 or undefined.
-  const height = gl.canvas ? gl.canvas.clientHeight || gl.canvas.height : 100;
+  const canvasHeight = gl.canvas ? gl.canvas.clientHeight || gl.canvas.height : 100;
+
+  // Default framebuffer is used when writing to canvas
+  const height = useTarget ? targetHeight : canvasHeight;
```

Comment:
This seems more complicated than it needs to be. You should be able to use 

```js
const height = useTarget ? target.height : gl.drawingBufferHeight;
```

Then

```js
height - (dimensions.y + dimensions.height) * pixelRatio
```

Reply:
Fixed